packages/api/src/routes/medical/patient.ts

```diff
@@ -33,8 +33,14 @@ import {
   schemaCreateToPatient,
   schemaUpdateToPatient,
 } from "./schemas/patient";
+import { createConsolidatedPatientData } from "../../command/medical/patient/create-consolidate-data";
+import { bundleSchema } from "./schemas/fhir";
+import { validateFhirEntries } from "../../external/fhir/shared/json-validator";
 
 const router = Router();
+const MAX_RESOURCE_POST_LIMIT = 50;
+const MAX_RESOURCE_STORED_LIMIT = 1000;
```

Comment:
`MAX` and `LIMIT` convey the same idea, maybe we could indicate the unit instead, like `MAX_RESOURCE_POST_COUNT` or similar?